### DISCUSSION TITLE
友链墙

### DISCUSSION THREAD

**Discussion by caibingcheng** [caibingcheng/comment#114] [Announcements]
https://bbing.com.cn/friends/

**Comment by penndu** [comment]
友链我加上了，请待CDN的缓存更新。

另外，要不要换个评论系统啊~

**Comment by caibingcheng** [comment]
> @penndu
> 友链我加上了，请待CDN的缓存更新。
> 
> 另外，要不要换个评论系统啊~

我是从waline切换过来的，暂没有计划换评论系统，近期不折腾了😅

**Comment by 7hr0wer** [comment]
你好，我是thrower.cc的站长，目前博客已恢复访问！

**Comment by caibingcheng** [comment]
> @7hr0wer
> 你好，我是thrower.cc的站长，目前博客已恢复访问！

收到，本站现在用的uptimerobot自动分类，恢复访问后，大约24h内会自动恢复对贵站的友链~

**Comment by ryan4yin** [comment]
换了域名的话，本页的「本站友链」内容可以更新下~

顺便我已经将友链地址更新为了 <https://imcbc.cn/>

**Comment by caibingcheng** [comment]
> @ryan4yin
> 换了域名的话，本页的「本站友链」内容可以更新下~
> 
> 顺便我已经将友链地址更新为了 <https://imcbc.cn/>

还没计划什么时候更换域名，两个域名都还可以用，还会继续续费

**Comment by panqiincs** [comment]
你好，我们的技术栈有点相似，请问可以交换链接吗？我这边已经添加，我的站点信息如下：

{
    "name":"辛未羊的网络日志",
    "url":"https://panqiiincs.me/",
    "logo":"https://panqiincs.me/images/avatar.jpg",
    "word":"记录生活和学习的清静之地"
}

**Comment by caibingcheng** [comment]
> @panqiincs
> 你好，我们的技术栈有点相似，请问可以交换链接吗？我这边已经添加，我的站点信息如下：
> 
> {
>     "name":"辛未羊的网络日志",
>     "url":"https://panqiiincs.me/",
>     "logo":"https://panqiincs.me/images/avatar.jpg",
>     "word":"记录生活和学习的清静之地"
> }

可以呀，但是我现在打不开你的网站

logo到是可以打开

**Comment by panqiincs** [comment]
> @caibingcheng
> > @panqiincs
> > 你好，我们的技术栈有点相似，请问可以交换链接吗？我这边已经添加，我的站点信息如下：
> > 
> > {
> >     "name":"辛未羊的网络日志",
> >     "url":"https://panqiiincs.me/",
> >     "logo":"https://panqiincs.me/images/avatar.jpg",
> >     "word":"记录生活和学习的清静之地"
> > }
> 
> 可以呀，但是我现在打不开你的网站
> 
> logo到是可以打开

托管在Github Pages上，在有的网络环境下偶尔会抽风，晚点再试试看吧。

**Comment by caibingcheng** [comment]
> @panqiincs
> 
> 托管在Github Pages上，在有的网络环境下偶尔会抽风，晚点再试试看吧。

原来是自控大佬( ఠൠఠ )ﾉ！已经添加了~可能要过会才更新上

**Comment by caibingcheng** [comment]
> @panqiincs
> > @caibingcheng
> > > @panqiincs
> > > 你好，我们的技术栈有点相似，请问可以交换链接吗？我这边已经添加，我的站点信息如下：
> > > 
> > > {
> > >     "name":"辛未羊的网络日志",
> > >     "url":"https://panqiiincs.me/",
> > >     "logo":"https://panqiincs.me/images/avatar.jpg",
> > >     "word":"记录生活和学习的清静之地"
> > > }
> > 
> > 可以呀，但是我现在打不开你的网站
> > 
> > logo到是可以打开
> 
> 托管在Github Pages上，在有的网络环境下偶尔会抽风，晚点再试试看吧。

原来是因为链接中的`panqiiincs.me`多打了一个`i` !

**Comment by panqiincs** [comment]
> @caibingcheng
> > @panqiincs
> > > @caibingcheng
> > > > @panqiincs
> > > > 你好，我们的技术栈有点相似，请问可以交换链接吗？我这边已经添加，我的站点信息如下：
> > > > 
> > > > {
> > > >     "name":"辛未羊的网络日志",
> > > >     "url":"https://panqiiincs.me/",
> > > >     "logo":"https://panqiincs.me/images/avatar.jpg",
> > > >     "word":"记录生活和学习的清静之地"
> > > > }
> > > 
> > > 可以呀，但是我现在打不开你的网站
> > > 
> > > logo到是可以打开
> > 
> > 托管在Github Pages上，在有的网络环境下偶尔会抽风，晚点再试试看吧。
> 
> 原来是因为链接中的`panqiiincs.me`多打了一个`i` !

两个i连在一起确实容易看漏，当年选域名时太草率了。

**Comment by panqiincs** [comment]
咦，我前天发的日志怎么不在友链文章里，你是写程序自动抓取的吗？

**Comment by caibingcheng** [comment]
> @panqiincs
> 咦，我前天发的日志怎么不在友链文章里，你是写程序自动抓取的吗？

相当于通过rss自动抓取的，可能因为没有添加你的订阅😅（因为没有找到...），现在找到了，我添加一下(更新可能要六个小时⏰)

**Comment by iP3ter** [comment]
{
    "name":"Peter's Blog",
    "url":"https://www.rainogo.com/",
    "logo":"https://bu.dusays.com/2022/06/21/62b1583017bae.jpg",
    "word":"来人间一趟，你要看看太阳"
}
已经添加你的了

**Comment by caibingcheng** [comment]
> @Rainogo
> {
>     "name":"Peter's Blog",
>     "url":"https://www.rainogo.com/",
>     "logo":"https://bu.dusays.com/2022/06/21/62b1583017bae.jpg",
>     "word":"来人间一趟，你要看看太阳"
> }
> 已经添加你的了

你好，已经更新了~~~（居然过去五天了(○´･д･)ﾉ

**Comment by iP3ter** [comment]
原Peter's Blog网址更换了
更换为 https://p3ter.me/
P3ter's Blog
麻烦了😘

**Comment by iP3ter** [comment]
原P3ter's Blog网址更换了
更换为 https://p3ter.me/
P3ter's Blog
麻烦了😘

**Reply by iP3ter** [reply]
一不小心多发了一条（

**Reply by caibingcheng** [reply]
已更新... 但是 https://p3ter.me/ 这个网址不可访问了吗😰

**Reply by caibingcheng** [reply]
现在访问正常了

**Comment by yunyit** [comment]
大佬，已添加贵站链接了，谢谢😊
```
{
    "name":"Yunyi's Blog",
    "url":"https://www.yunyitang.me/zh/",
    "logo":"https://www.yunyitang.me/img/Avatar.png",
    "word":"Little squirrel Hopping around"
}
```

**Reply by caibingcheng** [reply]
已添加

**Reply by yunyit** [reply]
前阵子忙秃头疏于更新博客，现在恢复正常更新了～～

**Comment by nxyxs** [comment]
网站名称：秃头张
网站描述：探索世界，发现自己
网站地址：https://www.tutouzhang.com
网站头像：https://www.nxyxs.cn/bloglogo.jpg

已添加贵站友链，还请大佬给个位置

**Reply by nxyxs** [reply]
我的还没加

**Reply by caibingcheng** [reply]
已添加，但是现在似乎无法访问

**Reply by nxyxs** [reply]
可以访问了

**Comment by yunyit** [comment]
新年快乐！我的域名改成 https://www.yunyitang.me 啦（去掉了/zh/）

**Reply by caibingcheng** [reply]
新年快乐(〃'▽'〃)   已更新

**Reply by nxyxs** [reply]
大佬 我的还没加

**Comment by nxyxs** [comment]
oos速度有点慢 现在正在迁移 很快就好

--------------原始邮件--------------
发件人："Bing ***@***.***&gt;;
发送时间：2024年6月5日(星期三) 下午4:44
收件人："caibingcheng/comment" ***@***.***&gt;;
抄送："张昊 ***@***.***&gt;;"Comment ***@***.***&gt;;
主题：Re: [caibingcheng/comment] 友链墙 (Discussion #114)
-----------------------------------

 

 
已添加，但是现在似乎无法访问

**Comment by Kings-en** [comment]
{
    "name":"AONOTE",
    "url":"https://www.aonote.cn",
    "logo":"https://www.aonote.cn/img/avatar.jpg",
    "word":"SOMETIMES CODE, SOMETIMES DESIGN"
}
已添加贵站友链，还请大佬给个位置

**Reply by caibingcheng** [reply]
已添加

**Comment by seminorm** [comment]
您好，很感谢添加了本站(https://hugo.bnblogs.cc)的友链，由于服务器到期，本站即将不可访问，上线时间不定，如果介意请注意移除本站友链，后续上线后我会再次申请友链！

**Comment by panqiincs** [comment]
刚发现，我的网站在你这里是全站链接，我回去也给你加一个全站链接。

**Comment by liangbm3** [comment]
{
    "name":"liangbm3's blog",
    "url":"https://liangbm3.top/",
    "logo":"https://liangbm3.top/upload/avatar.png",
    "word":"总有人间一两风，填我十万八千梦！"
}

**Comment by panqiincs** [comment]
ImCBC，你好！「辛未羊的网络日志」现已更新域名和站名，原域名已设置永久跳转。相关信息如下，烦请有空时帮忙更新，谢谢！

站名：北州店子
链接：https://numden.com/
简介：记录生活和学习
头像：https://numden.com/images/avatar.jpg
订阅：https://numden.com/atom.xml
